examples/src/test/java/ai/djl/examples/inference/ImageClassificationTest.java

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
We don't need this test, we have the same test in TrainMnistTest.java

Reply:
OK, I miss that. I will delete it.

Reply:
Please add check the value should be less than 1 in TrainMnistTest.java

Reply:
I'm a little confused about the if statement `Boolean.getBoolean("nightly")`. When does it become true?

Reply:
We have continuous and nightly build, see:
https://github.com/deepjavalibrary/djl/blob/master/.github/workflows/nightly_publish.yml#L37